app/src/main/res/layout/dismissible_page_no_title_fragment.xml

```diff
@@ -8,26 +9,28 @@
 
     <ImageView
         android:id="@+id/illustration"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:layout_gravity="center_horizontal"
-        android:layout_marginStart="@dimen/base_margin_triple"
-        android:layout_marginEnd="@dimen/base_margin_triple"
-        android:layout_weight="1"
+        android:layout_width="@dimen/dismissible_page_illustration_size"
+        android:layout_height="@dimen/dismissible_page_illustration_size"
         android:contentDescription="@null"
+        android:maxHeight="200dp"
```

Comment:
extract to dimens

Reply:
actually come to think of it, is this necessary? given that we have a fixed width/height.

Reply:
Ture! I forgot to remove it